lr_face/models.py

```diff
@@ -0,0 +1,45 @@
+import numpy as np
+
+from scipy import spatial
+from lr_face.utils import resize
+
+
+class DummyModel:
+    """
+    dummy model that return random scores
+    """
+
+    def __init__(self, resolution=(100, 100)):
+        self.resolution = resolution
+
+    def fit(self, X, y):
+        assert X.shape[1:3] == self.resolution
+        pass
+
+    def predict_proba(self, X):
+        assert X.shape[2:4] == self.resolution
```

Comment:
wil je dit hard eisen? zou m uitcommenten

Reply:
Dit is jouw code ;) ik wil hem best uitcommenten.